### DISCUSSION TITLE
Version to run in Java 8 and JBoss EAP 7.5

### DISCUSSION THREAD

**Discussion by alfarafaelm** [primefaces/community#2565] [PrimeFaces] [answered]
Hi,

Is there a specific version of Freya template to run in an environment of Java 8 and JBoss EAP 7.5?

**Comment by jepsar** [comment, marked as answer]
Templates are linked to certain version of PrimeFaces. PrimeFaces is compatible with Java 8 up to version 13.

https://github.com/primefaces/primefaces?tab=readme-ov-file#versions
